recipes/theora/all/conandata.yml

```diff
@@ -1,4 +1,6 @@
 sources:
   "1.1.1":
-    url: "http://downloads.xiph.org/releases/theora/libtheora-1.1.1.zip"
-    sha256: "f644fef154f7a80e7258c8baec5c510f594d720835855cddce322b924934ba36"
+    - url: "http://downloads.xiph.org/releases/theora/libtheora-1.1.1.zip"
```

Comment:
```suggestion
    - url: "https://downloads.xiph.org/releases/theora/libtheora-1.1.1.zip"
```

Reply:
It seems xiph hasn't upgraded their servers for the modern SSL, we are forced to use http...